src/libraries/ComparorUint256.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.13;
+import "ROOT/libraries/DelegationTarget.sol";
+
+
+contract ComparorUint256 is DelegationTarget {
+
```

Comment:
Style nit: Empty lines after the contract definition and before it closes

Reply:
Hmm... this file should have been deleted...

Reply:
Fixed via deletion of file.